src/nunit.uikit/MessageDisplay.cs

```diff
@@ -49,25 +49,45 @@ public MessageDisplay(string caption)
 
         public DialogResult Display(string message)
         {
-            return Display(message, MessageBoxButtons.OK);
+            return Display(message, _defaultCaption, MessageBoxButtons.OK);
+        }
+
+        public DialogResult Display(string message, string caption)
+        {
+            return Display(message, caption, MessageBoxButtons.OK);
         }
 
         public DialogResult Display(string message, MessageBoxButtons buttons)
         {
+            return MessageBox.Show(message, _defaultCaption, buttons);
```

Comment:
This could just call the overload Display to match previous 2 methods. Same applies to Error Info and Ask.